web-api/workflow-terraform/migration/main/lambdas/migrations/0010-add-lead-docket-number-to-case-deadline.ts

```diff
@@ -0,0 +1,39 @@
+import { CaseDeadline } from '../../../../../../shared/src/business/entities/CaseDeadline';
+import { aggregateCaseItems } from '../../../../../../shared/src/persistence/dynamo/helpers/aggregateCaseItems';
+import { createApplicationContext } from '../../../../../src/applicationContext';
+import { queryFullCase } from '../utilities/queryFullCase';
+
+const applicationContext = createApplicationContext({});
+
+const isCaseDeadlineItem = item => {
+  return (
+    item.pk.startsWith('case-deadline|') && item.sk.startsWith('case-deadline|')
+  );
+};
+
+export const migrateItems = async (items, documentClient) => {
+  const itemsAfter: any[] = [];
```

Comment:
Same here

Reply:
Good call.  I did not realize the CaseDeadline class could actually be used as the type.